cfme/utils/appliance/__init__.py

```diff
@@ -1035,12 +1035,11 @@ def miqqe_patch_applied(self):
 
     @logger_wrap("Patch appliance with MiqQE js: {}")
     def patch_with_miqqe(self, log_callback=None):
-
         # (local_path, remote_path, md5/None) trio
-        autofocus_patch = pick({
+        autofocus_patch = VersionPicker({
             '5.5': 'autofocus.js.diff',
             '5.7': 'autofocus_57.js.diff'
-        })
```

Comment:
we can remove 5.5 and 5.7 version picks now

Reply:
Actually, this method is not needed anymore, but it should be removed in a next PR.

Reply:
:+1: